source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -105,6 +105,9 @@ OpenGLContext: class extends GpuContext {
 	_searchImageBin: func (type: Class, size: IntVector2D) -> GpuImage {
 		this _recycleBin search(|image| image instanceOf(type) && image size == size)
 	}
+	createFence: override func -> Promise {
+		OpenGLPromise new(this)
```

Comment:
Should we sync the fence here as well, then we don't need to cast to OpenGLPromise